invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/setup.py

```diff
@@ -32,6 +32,8 @@
     },
     install_requires=[
```

Comment:
BTW instead of `cookiecutter: ...`, the commit headline should rather be `installation: ...`.